Firestore/core/test/unit/local/remote_document_cache_test.cc

```diff
@@ -98,7 +98,7 @@ MATCHER_P(HasAtLeastDocs,
 }
 
 MATCHER_P(MatchesValue, expected, "") {
-  const google_firestore_v1_Value& actual_value = arg.data().Get();
+  const google_firestore_v1_Value& actual_value = arg.value();
```

Comment:
Nit: it's best not to bind references to temporaries without need (I think we discussed the rationale at some point, so I'll omit it here, but I'm happy to talk about it if you'd like).

Reply:
Done. I do remember the advice (at least for now...).